addons/web_editor/static/src/js/wysiwyg/plugin/keyboard.js

```diff
@@ -998,7 +994,47 @@ var KeyboardPlugin = AbstractPlugin.extend({
         }
         return range;
     },
+    /**
+     * create a container to focus and who receive the accented char or the
+     * auto-completion string.
+     *
+     * @private
+     * @param {function} callback
+     */
+    _onAccentedAndVirtualKey: function (callback) {
+        var self = this;
+        if (this._onAccentedAndVirtualKeyCallback) {
+            return;
+        }
+
+        this.editable.normalize();
+        var baseRange = this.context.invoke('editor.createRange');
 
+        var $parent = $(baseRange.sc.parentNode);
+        var parentContenteditable = $parent.attr('contenteditable');
+        $parent.attr('contenteditable', false);
+
+        var accentPlaceholder = this.document.createElement('span');
+        accentPlaceholder.style.position = 'absolute';
+        accentPlaceholder.style.zIndex = '-100';
+        $(baseRange.sc).after(accentPlaceholder);
+        $(accentPlaceholder).attr('contenteditable', true);
+
+        this.context.invoke('editor.setRange', accentPlaceholder, 0).select();
+
+        this._onAccentedAndVirtualKeyCallback = function () {
+            var text = accentPlaceholder.textContent;
+            if (parentContenteditable) {
+                $parent.attr('contenteditable', parentContenteditable);
+            } else {
+                $parent.removeAttr('contenteditable');
+            }
+            $(accentPlaceholder).remove();
+            callback(text, baseRange);
+            this._onAccentedAndVirtualKeyCallback = null;
+        };
+        this._onAccentedAndVirtualKeyTimer = setTimeout(this._onAccentedAndVirtualKeyCallback.bind(this));
```

Comment:
This setTimeout seems dangerous, the event stack might be flooded by touch/scroll events